bindings/rust/build.rs

```diff
@@ -155,9 +156,6 @@ fn replace_ckzg_ret_repr(mut bindings: String) -> String {
         .rfind(repr_to_replace)
         .expect("Could not find repr to replace in bindings");
 
-    // Sanity check that it's an attribute of `C_KZG_RET` and not another type.
-    assert!(repr_start > bindings[..ckzg_ret].rfind('}').unwrap());
```

Comment:
Just to note, removed this because it doesn't work if `C_KZG_RET` is the first type.